OrbitClientGgp/ClientGgp.cpp

```diff
@@ -234,16 +242,39 @@ void ClientGgp::OnCaptureStarted(
   LOG("Capture started");
 }
 
-void ClientGgp::OnCaptureComplete() { LOG("Capture completed"); }
+void ClientGgp::OnCaptureComplete() {
+  LOG("Capture completed");
+  SamplingProfiler sampling_profiler(*capture_data_.GetCallstackData(), capture_data_);
+  capture_data_.set_sampling_profiler(sampling_profiler);
+}
 
-void ClientGgp::OnTimer(const orbit_client_protos::TimerInfo&) {}
+void ClientGgp::OnTimer(const orbit_client_protos::TimerInfo& timer_info) {
+  if (timer_info.function_address() > 0) {
+    FunctionInfo* func =
+        capture_data_.process()->GetFunctionFromAddress(timer_info.function_address());
+    CHECK(func != nullptr);
```

Comment:
Is the Check-fail intended here? I briefly looked into GetFunctionFromAddress and it seems returning nullptr is not that far fetched. 
If all these cases are "impossible" (Is this the case? We only can get samples with symbols from our instrumentation?) add a brief comment in front of the CHECK.

Reply:
For function timers, the function *must* be present in the process. If this is not the case, our data model has changed, thus failing is good here.

Reply:
I'm gonna add a brief comment before the check to clarify it, thanks!

Reply:
Done!

Reply:
Thanks!